db/internal_stats.h

```diff
@@ -307,6 +341,20 @@ class InternalStats {
     uint64_t ingest_l0_files_addfile;  // Total number of files ingested to L0
     uint64_t ingest_keys_addfile;      // Total number of keys ingested
 
+    void Clear() {
```

Comment:
Move this Clear() function and the one in DBStatsSnapshot to be after the constructors (as constructors should almost always come first).